Aikindapps/Nuance
# Implemented middleware functions to fetch the NUA transactions and ck…

…BTC transactions. Displayed them in the wallet screen.

How to test:
Locally, you can deposit NUA or ckBTC to your local principal id in the wallet screen. You should see the deposit transaction. But please don't forget you'll lose the sent tokens if you want to try it locally :)
